### PR TITLE
Fix profile: ffmpeg

### PR DESCRIPTION
**Pull request by the-antz**
Fix broken libx265 encoding (needs the set_mempolicy syscall).

### CODE REVIEW

File: etc/ffmpeg.profile [hunk 1]
```diff
@@ -38,7 +38,8 @@ notv
 nou2f
 novideo
 protocol inet,inet6
-seccomp
+# drop @default, except set_mempolicy, which is required to encode using libx265
+seccomp.drop _sysctl,acct,add_key,adjtimex,afs_syscall,bdflush,bpf,break,chroot,clock_adjtime,clock_settime,create_module,delete_module,fanotify_init,finit_module,ftime,get_kernel_syms,getpmsg,gtty,init_module,io_cancel,io_destroy,io_getevents,io_setup,io_submit,ioperm,iopl,ioprio_set,kcmp,kexec_file_load,kexec_load,keyctl,lock,lookup_dcookie,mbind,migrate_pages,modify_ldt,mount,move_pages,mpx,name_to_handle_at,nfsservctl,ni_syscall,open_by_handle_at,pciconfig_iobase,pciconfig_read,pciconfig_write,perf_event_open,personality,pivot_root,process_vm_readv,process_vm_writev,prof,profil,ptrace,putpmsg,query_module,reboot,remap_file_pages,request_key,rtas,s390_mmio_read,s390_mmio_write,s390_runtime_instr,security,setdomainname,sethostname,settimeofday,sgetmask,ssetmask,stime,stty,subpage_prot,swapoff,swapon,switch_endian,sys_debug_setcontext,sysfs,syslog,tuxcall,ulimit,umount,umount2,uselib,userfaultfd,ustat,vhangup,vm86,vm86old,vmsplice,vserver
```
Comment (rusty-snake) [inline etc/ffmpeg.profile]:
```suggestion
seccomp !set_mempolicy
```

File: etc/ffmpeg.profile [hunk 2]
```diff
@@ -38,7 +38,8 @@ notv
 nou2f
 novideo
 protocol inet,inet6
-seccomp
+# drop @default, except set_mempolicy, which is required to encode using libx265
```
Comment (rusty-snake) [inline etc/ffmpeg.profile]:
```suggestion
# allow set_mempolicy, which is required to encode using libx265
```

**Comment by the-antz** [comment]
Hi,
I pushed the suggested changes.

Didn't know about the exceptions syntax but I missed it when editing the file. Great to hear it already exists.

**Comment by rusty-snake** [comment]
Merged, Thanks.

**Pull request merged**